2025-08-XX-qubes-os-4-3-0-rc1-available-for-testing.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Qubes OS 4.3.0-rc1 is available for testing"
+categories: releases
+download_url: /downloads/
+---
+
+We're pleased to announce that the first release candidate (RC) for Qubes OS 4.3.0 is now available for testing. This minor release includes several new features and improvements over Qubes OS 4.2. The ISO and associated [verification files](/security/verifying-signatures/) are available on the [downloads](/downloads/) page.
+
+## What's new in Qubes 4.3.0?
+
+- Dom0 upgraded to Fedora 41 ([#9402](https://github.com/QubesOS/qubes-issues/issues/9402)).
+- Xen upgraded to version 4.19 ([#9420](https://github.com/QubesOS/qubes-issues/issues/9420)).
+- Default Fedora template upgraded to Fedora 41 (older releases not supported).
+- Default Debian template upgraded to Debian 12 (older releases not supported).
+- Default Whonix templates upgraded to Whonix 17.4.3 (older releases not supported).
+- Preloaded disposables ([#1512](https://github.com/QubesOS/qubes-issues/issues/1512)
+- Device "self-identity oriented" assignment (a.k.a. New Devices API) ([#9325](https://github.com/QubesOS/qubes-issues/issues/9325))
+- QWT (Qubes Windows Tools) reintroduced with improved features ([#1861](github.com/QubesOS/qubes-issues/issues/1861)).
+
+For more information about the changes included in this version, see the [Qubes OS 4.3.0 release notes](/doc/releases/4.3/release-notes/).
+
+## When is the stable release?
+
+That depends on the number of bugs discovered in this RC and their severity. As explained in our [release schedule](/doc/version-scheme/#release-schedule) documentation, our usual process after issuing a new RC is to collect bug reports, triage the bugs, and fix them. If warranted, we then issue a new RC that includes the fixes and repeat the process. We continue this iterative procedure until we're left with an RC that's good enough to be declared the stable release. No one can predict, at the outset, how many iterations will be required (and hence how many RCs will be needed before a stable release), but we tend to get a clearer picture of this as testing progresses.
+
+## Testing Qubes 4.3.0-rc1
+
+If you're willing to [test](/doc/testing/) this new RC, you can help us improve the eventual stable release by [reporting any bugs you encounter](/doc/issue-tracking/). We encourage experienced users to join the [testing team](https://forum.qubes-os.org/t/joining-the-testing-team/5190). The best way to test Qubes 4.3.0-rc1 is by performing a [clean installation](/doc/installation-guide/) with the new ISO. We strongly recommend [making a full backup](/doc/how-to-back-up-restore-and-migrate/) beforehand. (Please note that [in-place upgrades from 4.2 to 4.3](https://github.com/QubesOS/qubes-issues/issues/9317) have not yet been implemented as of RC1.)
```

Comment:
In-place upgrade has been added, may need --enable-current-testing option (see comment in the release notes PR)